2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten] MongoDB starting : pid=13496 port=27017 dbpath=/home/simonx/Documents/Udacity/Projects/capstone/PESM_NSGA2_CST_XFoil/datadir/ 64-bit host=simonx
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten] db version v3.4.4
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten] git version: 888390515874a9debd1b6c5d36559ca86b44babd
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.1f 6 Jan 2014
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten] allocator: tcmalloc
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten] modules: none
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten] build environment:
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten]     distmod: ubuntu1404
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten]     distarch: x86_64
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten]     target_arch: x86_64
2017-04-29T17:52:40.875+0200 I CONTROL  [initandlisten] options: { processManagement: { fork: true }, storage: { dbPath: "datadir/" }, systemLog: { destination: "file", path: "logfile" } }
2017-04-29T17:52:40.917+0200 I -        [initandlisten] Detected data files in /home/simonx/Documents/Udacity/Projects/capstone/PESM_NSGA2_CST_XFoil/datadir/ created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2017-04-29T17:52:40.917+0200 I STORAGE  [initandlisten] 
2017-04-29T17:52:40.917+0200 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2017-04-29T17:52:40.917+0200 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2017-04-29T17:52:40.918+0200 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=11495M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),checkpoint=(wait=60,log_size=2GB),statistics_log=(wait=0),
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] 
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] 
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] 
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] 
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/defrag is 'always'.
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2017-04-29T17:52:41.366+0200 I CONTROL  [initandlisten] 
2017-04-29T17:52:41.371+0200 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/home/simonx/Documents/Udacity/Projects/capstone/PESM_NSGA2_CST_XFoil/datadir/diagnostic.data'
2017-04-29T17:52:41.372+0200 I NETWORK  [thread1] waiting for connections on port 27017
2017-04-29T17:52:41.834+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38052 #1 (1 connection now open)
2017-04-29T17:52:41.834+0200 I NETWORK  [conn1] received client metadata from 127.0.0.1:38052 conn1: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:52:41.835+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38054 #2 (2 connections now open)
2017-04-29T17:52:41.835+0200 I NETWORK  [conn2] received client metadata from 127.0.0.1:38054 conn2: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:53:14.525+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38056 #3 (3 connections now open)
2017-04-29T17:53:14.525+0200 I NETWORK  [conn3] received client metadata from 127.0.0.1:38056 conn3: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:53:14.526+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38058 #4 (4 connections now open)
2017-04-29T17:53:14.526+0200 I NETWORK  [conn4] received client metadata from 127.0.0.1:38058 conn4: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:53:17.655+0200 I -        [conn3] end connection 127.0.0.1:38056 (4 connections now open)
2017-04-29T17:53:17.655+0200 I -        [conn4] end connection 127.0.0.1:38058 (4 connections now open)
2017-04-29T17:53:51.291+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38064 #5 (3 connections now open)
2017-04-29T17:53:51.291+0200 I NETWORK  [conn5] received client metadata from 127.0.0.1:38064 conn5: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:53:51.292+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38066 #6 (4 connections now open)
2017-04-29T17:53:51.292+0200 I NETWORK  [conn6] received client metadata from 127.0.0.1:38066 conn6: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:53:56.466+0200 I -        [conn6] end connection 127.0.0.1:38066 (4 connections now open)
2017-04-29T17:53:56.466+0200 I -        [conn5] end connection 127.0.0.1:38064 (4 connections now open)
2017-04-29T17:54:31.948+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38070 #7 (3 connections now open)
2017-04-29T17:54:31.949+0200 I NETWORK  [conn7] received client metadata from 127.0.0.1:38070 conn7: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:54:31.949+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38072 #8 (4 connections now open)
2017-04-29T17:54:31.949+0200 I NETWORK  [conn8] received client metadata from 127.0.0.1:38072 conn8: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:54:35.347+0200 I -        [conn8] end connection 127.0.0.1:38072 (4 connections now open)
2017-04-29T17:54:35.347+0200 I -        [conn7] end connection 127.0.0.1:38070 (4 connections now open)
2017-04-29T17:55:19.956+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38076 #9 (3 connections now open)
2017-04-29T17:55:19.956+0200 I NETWORK  [conn9] received client metadata from 127.0.0.1:38076 conn9: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:55:19.958+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38078 #10 (4 connections now open)
2017-04-29T17:55:19.958+0200 I NETWORK  [conn10] received client metadata from 127.0.0.1:38078 conn10: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:55:24.319+0200 I -        [conn10] end connection 127.0.0.1:38078 (4 connections now open)
2017-04-29T17:55:24.319+0200 I -        [conn9] end connection 127.0.0.1:38076 (4 connections now open)
2017-04-29T17:56:01.933+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38080 #11 (3 connections now open)
2017-04-29T17:56:01.933+0200 I NETWORK  [conn11] received client metadata from 127.0.0.1:38080 conn11: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:56:01.934+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38082 #12 (4 connections now open)
2017-04-29T17:56:01.934+0200 I NETWORK  [conn12] received client metadata from 127.0.0.1:38082 conn12: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:56:05.640+0200 I -        [conn12] end connection 127.0.0.1:38082 (4 connections now open)
2017-04-29T17:56:05.640+0200 I -        [conn11] end connection 127.0.0.1:38080 (4 connections now open)
2017-04-29T17:56:43.646+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38084 #13 (3 connections now open)
2017-04-29T17:56:43.646+0200 I NETWORK  [conn13] received client metadata from 127.0.0.1:38084 conn13: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:56:43.648+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38086 #14 (4 connections now open)
2017-04-29T17:56:43.648+0200 I NETWORK  [conn14] received client metadata from 127.0.0.1:38086 conn14: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:56:47.570+0200 I -        [conn14] end connection 127.0.0.1:38086 (4 connections now open)
2017-04-29T17:56:47.570+0200 I -        [conn13] end connection 127.0.0.1:38084 (4 connections now open)
2017-04-29T17:57:22.470+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38088 #15 (3 connections now open)
2017-04-29T17:57:22.471+0200 I NETWORK  [conn15] received client metadata from 127.0.0.1:38088 conn15: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:57:22.472+0200 I NETWORK  [thread1] connection accepted from 127.0.0.1:38090 #16 (4 connections now open)
2017-04-29T17:57:22.472+0200 I NETWORK  [conn16] received client metadata from 127.0.0.1:38090 conn16: { driver: { name: "PyMongo", version: "3.4.0" }, os: { type: "Linux", name: "Ubuntu 14.04 trusty", architecture: "x86_64", version: "4.4.0-72-generic" }, platform: "CPython 2.7.12.final.0" }
2017-04-29T17:57:25.736+0200 I -        [conn16] end connection 127.0.0.1:38090 (4 connections now open)
2017-04-29T17:57:25.736+0200 I -        [conn15] end connection 127.0.0.1:38088 (4 connections now open)
2017-04-29T17:57:39.507+0200 I -        [conn2] end connection 127.0.0.1:38054 (2 connections now open)
2017-04-29T17:57:39.507+0200 I -        [conn1] end connection 127.0.0.1:38052 (2 connections now open)
